Client/Frontend/Strings.swift

```diff
@@ -243,10 +243,10 @@ extension String {
                 value: "Done",
                 comment: "When a user is in the process of making a purchase and has at least one saved credit card, a view above the keyboard shows actions a user can take. When tapping this label, the keyboard will dismiss from view.")
             public static let ListItemA11y = MZLocalizedString(
-                key: "CreditCard.Settings.ListItemA11y.v112",
+                key: "CreditCard.Settings.ListItemA11y.v118",
                 tableName: "Settings",
-                value: "%1$@ ending in %2$@, issued to %3$@, expires %4$@",
-                comment: "Accessibility label for a credit card list item in autofill settings screen. The first parameter is the credit card type (e.g. Visa). The second parameter is the last 4 digits of the credit card. The third parameter is the name of the credit card holder. The fourth and fifth parameters are the month and year of the credit card's expiration date.")
+                value: "%1$@, issued to %2$@, ending in %3$@, expires %4$@",
```

Comment:
Was this change discussed with Emily? The original version was coming from her and was approved.

Reply:
no and I was not aware of that, only of the existence of this task that has been in the to do spreadsheet for credit cards for a few weeks.

Reply:
Then I suggest you keep the original string as it's approved and just adjust the order of the strings used with it when creating the voice over string.

Reply:
by reading the case study in the ticket I'd rather say that Emily's approved string is not optimal for the users that will benefit from this change. Having a different order in the implementation would cause translations to not make sense in several languages because of connector words and could cause confusion when handling this area of the code.

Reply:
@petruSt24 we have experts to create the optimal strings. If a string needs to be changed and the new string is not approved yet (not defined in the ticket), you should consult with the experts before changing it.

Reply:
@thatswinnie awesome, then I'll text Emily about this

Reply:
@thatswinnie 
![image](https://github.com/mozilla-mobile/firefox-ios/assets/126153378/1fbf8db9-2232-4565-a9ca-74641b4cc63e)